x-pack/plugins/secops/public/components/page/hosts/host_summary/index.test.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { Provider as ReduxStoreProvider } from 'react-redux';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+
+import { mockGlobalState } from '../../../../mock';
+import { createStore, State } from '../../../../store';
+import { DragDropContextWrapper } from '../../../drag_and_drop/drag_drop_context_wrapper';
+import { getEmptyValue } from '../../../empty_value';
+import { createDraggable, getEuiDescriptionList, HostSummary } from './index';
+import { mockData } from './mock';
+
+describe('Host Summary Component', () => {
+  const state: State = mockGlobalState;
+
+  let store = createStore(state);
+
+  beforeEach(() => {
+    store = createStore(state);
+  });
+
+  describe('rendering', () => {
+    test('it renders the default Host Summary', () => {
+      const wrapper = shallow(
+        <ReduxStoreProvider store={store}>
+          <HostSummary
+            loading={false}
+            data={mockData.Hosts.edges}
+            startDate={552204000000}
+            endDate={618472800000}
+          />
+        </ReduxStoreProvider>
+      );
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+  });
+
+  describe('#createDraggable', () => {
+    test('if it creates a draggable component', () => {
+      const draggable = createDraggable(
+        'debian',
+        'Platform',
+        552204000000,
```

Comment:
I would make a test constant for this date time and probably do this

```ts
new Date("1987-07-02T06:00:00.000Z").valueOf()
```

So the person reading knows the human readable form the date times.

Reply:
That's fair -- in this instance those start/end times are irrelevant as it's for the draggable query which isn't tested at this level. I'll be sure to use human readable dates going forward.